src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -20,21 +20,31 @@
 public class LogsCenter {
     private static final int MAX_FILE_COUNT = 5;
     private static final int MAX_FILE_SIZE_IN_BYTES = (int) (Math.pow(2, 20) * 5); // 5MB
+    private static final String BASE_LOGGER_NAME = "AB3";
     private static final String LOG_FILE = "addressbook.log";
     private static Level currentLogLevel = Level.INFO;
     private static final Logger logger = LogsCenter.getLogger(LogsCenter.class);
     private static FileHandler fileHandler;
     private static ConsoleHandler consoleHandler;
+    private static final Logger baseLogger = Logger.getLogger(BASE_LOGGER_NAME);
 
     /**
-     * Initializes with a custom log level (specified in the {@code config} object)
-     * Loggers obtained *AFTER* this initialization will have their logging level changed<br>
-     * Logging levels for existing loggers will only be updated if the logger with the same name
-     * is requested again from the LogsCenter.
+     * Initializes with a custom log level (specified in the {@code config} object). All Loggers
+     * created using {@link #getLogger(String)} and {@link #getLogger(Class)} will have the same log
+     * level as the one specified in the {@code config} object.
      */
     public static void init(Config config) {
         currentLogLevel = config.getLogLevel();
         logger.info("currentLogLevel: " + currentLogLevel);
+        baseLogger.setLevel(currentLogLevel);
+
+        // Set the log level of log handlers to the current log level if they are not null
+        if (fileHandler != null) {
+            fileHandler.setLevel(currentLogLevel);
+        }
+        if (consoleHandler != null) {
+            consoleHandler.setLevel(currentLogLevel);
+        }
     }
```

Comment:
```java
    /**
     * Creates a logger with the given name.
     */
    public static Logger getLogger(String name) {
        Logger logger = Logger.getLogger(name);
        logger.setUseParentHandlers(false);
        removeHandlers(logger);
        addConsoleHandler(logger);
        addFileHandler(logger);
        return Logger.getLogger(name);
    }
```

Not related to this PR: In the code above (from this class), why do we use `return Logger.getLogger(name);` instead of `return logger;`?

Reply:
Logger itself statically maintain an mapping of all Loggers created. So returning logger and Logger.getLogger(name) returns the same thing.

Reply:
Yes, I get that it is the same thing. But `return logger;` is both shorter and more natural, right?

Reply:
@Eclipse-Dominator Can you do another small PR to fix this?

Reply:
It's not related to this PR at all. It will be confusing for reviewers.